src/asm/CMakeLists.txt

```diff
@@ -3,10 +3,12 @@ project(CemuAsm C)
 IF (WIN32)
 
 enable_language(C ASM_MASM)
+set(CMAKE_ASM_MASM_CREATE_STATIC_LIBRARY "<CMAKE_AR> /OUT:<TARGET> <LINK_FLAGS> <OBJECTS>")
```

Comment:
Since this is a workaround please add a comment explaining what's it for and make it conditional to the affected environment

Reply:
added